src/routers/user.ts

```diff
@@ -41,15 +45,17 @@ userRouter.put('/', async (req, res) => {
 		console.error(`Error: ${e} - Status Code ${res.statusCode}`);
 	}
 });
+*/
```

Comment:
there isn't any spec for updating a user in the new schema. do we still want this router?